Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -945,18 +1025,6 @@ public async Task OnPlaybackStopped(PlaybackStopInfo info)
                 }
             }
 
-            if (!string.IsNullOrEmpty(info.LiveStreamId))
```

Comment:
Seems like maybe this should be extracted to a helper method, not copy-pasted in a couple places.